poetry.lock

```diff
@@ -39,11 +27,11 @@ python-versions = ">=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*, !=3.4.*"
 
 [[package]]
 name = "commitizen"
-version = "2.20.0"
+version = "2.23.0"
 description = "Python commitizen client tool"
 category = "dev"
 optional = false
-python-versions = ">=3.6.1,<4.0.0"
+python-versions = ">=3.6.2,<4.0.0"
```

Comment:
nit(? did I do that right?) - this upgrade is not listed in the description

Reply:
I imagine this is just understood to fall withing "all transitive dependencies"?

Reply:
In general, you are correct that the modifications to `poetry.lock` typically fall in the transitive dependency category, although the specific line you noted actually details the supported versions of Python for one of our direct dependencies, commitizen. The lock file is managed automatically by Poetry, our Python dependency manager, so I modified `pyproject.toml` and then ran `poetry update` to generate these changes. One way of validating the changes to `poetry.lock` is to repeat that process locally and confirm that the resulting lock files are identical, but our pre-commit hooks actually already validate the lock file in CI, so testing it again locally typically isn't expected of a reviewer.

Reply:
Yeah, flagging nits vs. other feedback isn’t really a standardized thing, but your way is clear.